src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExample.java

```diff
@@ -323,34 +332,35 @@ static KafkaStreams createChartsStreams(final String bootstrapServers,
     // store "top-five-songs-by-genre", and this state store can then be queried interactively via a REST API (cf.
     // MusicPlaysRestService) for the latest charts per genre.
     songPlayCounts.groupBy((song, plays) ->
-            KeyValue.pair(song.getGenre().toLowerCase(),
-                new SongPlayCount(song.getId(), plays)),
-        Serdes.String(),
-        songPlayCountSerde)
-        // aggregate into a TopFiveSongs instance that will keep track
-        // of the current top five for each genre. The data will be available in the
-        // top-five-songs-genre store
-        .aggregate(TopFiveSongs::new,
-            (aggKey, value, aggregate) -> {
-              aggregate.add(value);
-              return aggregate;
-            },
-            (aggKey, value, aggregate) -> {
-              aggregate.remove(value);
-              return aggregate;
-            },
-            topFiveSerde,
-            TOP_FIVE_SONGS_BY_GENRE_STORE
-        );
+                                   KeyValue.pair(song.getGenre().toLowerCase(),
+                                                 new SongPlayCount(song.getId(), plays)),
+                           Serialized.with(Serdes.String(),
+                                           songPlayCountSerde))
+            // aggregate into a TopFiveSongs instance that will keep track
+            // of the current top five for each genre. The data will be available in the
+            // top-five-songs-genre store
+            .aggregate(TopFiveSongs::new,
+                       (aggKey, value, aggregate) -> {
+                         aggregate.add(value);
+                         return aggregate;
+                       },
+                       (aggKey, value, aggregate) -> {
+                         aggregate.remove(value);
+                         return aggregate;
+                       },
+                       Materialized.<String, TopFiveSongs, KeyValueStore<Bytes, byte[]>>as(TOP_FIVE_SONGS_BY_GENRE_STORE)
+                               .withKeySerde(Serdes.String())
+                               .withValueSerde(topFiveSerde)
+            );
 
     // Compute the top five chart. The results of this computation will continuously update the state
     // store "top-five-songs", and this state store can then be queried interactively via a REST API (cf.
     // MusicPlaysRestService) for the latest charts per genre.
     songPlayCounts.groupBy((song, plays) ->
             KeyValue.pair(TOP_FIVE_KEY,
                 new SongPlayCount(song.getId(), plays)),
-        Serdes.String(),
-        songPlayCountSerde)
+        Serialized.with(Serdes.String(),
```

Comment:
Again, is this auto-formatted?  Yes, it's a nit, but this line and the next should be on one line for better readability, and I wonder whether auto-formatting isn't set up correctly?  (Subsequent PRs might show spurious changes because formatting would change between versions.)

Reply:
Nit: make single line ?